cmd/node/main.go

```diff
@@ -1319,7 +1319,7 @@ func loadExternalConfig(filepath string) (*config.ExternalConfig, error) {
 	cfg := &config.ExternalConfig{}
 	err := core.LoadTomlFile(cfg, filepath)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("cannot load external config %w", err)
```

Comment:
separate the main error message and the resulted err.
`return nil, fmt.Errorf("cannot load external config: %w", err)` for example